akka-bench-jmh/src/main/scala/akka/stream/NewLayoutBenchmark.scala

```diff
@@ -17,8 +19,10 @@ import org.reactivestreams.{ Publisher, Subscriber, Subscription }
 @State(Scope.Benchmark)
 @OutputTimeUnit(TimeUnit.MILLISECONDS)
 @BenchmarkMode(Array(Mode.Throughput))
+ */
+@silent // too long to add pattern
```

Comment:
maybe change the comment to give a clue as to what the warning is

Reply:
It's a novel about a file with a package and nothing in it etc. Sure.